Docs: Installation refactor Windows (Split PR #5830) (#7669)
arangodb/arangodb: Documentation/Books/Manual/Installation/Windows.md

*&lt;ROOTDIR&gt;\etc\arangodb\*. Please adjust this to your needs if you want to
use different connection settings etc.

### Uninstalling

To uninstall the Arango server application you can use the windows control panel
(as you would normally uninstall an application). Note however, that any data
files created by the Arango server will remain as well as the *&lt;ROOTDIR&gt;*
directory.  To complete the uninstallation process, remove the data files and
the *&lt;ROOTDIR&gt;* directory manually.

Installing using the ZIP archive (XCopy installation)
-----------------------------------------------------

Not all users prefer the guided _Installer_ to install ArangoDB. In order to have a
[portable application](http://en.wikipedia.org/wiki/Portable_application), or easily
start different ArangoDB versions on the same machine, and/or for the maximum flexibility,
you might want to install using the _ZIP_ archive ([XCOPY deployment](http://en.wikipedia.org/wiki/XCOPY_deployment)).

### Unzip the archive

Open an explorer, choose a place where you would like ArangoDB to be, and extract the

> archive there. It will create its own toplevel directory with the version number in the string.

Author: ```suggestion
archive there. It will create its own top-level directory with the version number in the name.
```

### Edit the configuration

> This step is optional. If the default configuration of ArangoDB do not suite your needs,

Author: Maybe change this to:
```markdown
### Edit the configuration

*This step is optional.*

If the default configuration ...
```

Demo:

### Edit the configuration

*This step is optional.*

If the default configuration ...

Reviewer: done

> This step is optional. If the default configuration of ArangoDB do not suite your needs,

Author: Singular
```suggestion
This step is optional. If the default configuration of ArangoDB does not suite your needs,
```

> you might edit `etc\arangodb3\arangod.conf` to change some configuration options.

Author: ```suggestion
you can edit `etc\arangodb3\arangod.conf` to change or add configuration options.
```

### Start the Server

After installation, you may start ArangoDB in several ways. The exact start-up command

> depends on the type of ArangoDB deployment you are interested on (_Single Instance_, _Master-Slave_,

Author: ```suggestion
depends on the type of ArangoDB deployment you are interested in
```

> _Active Failover_ or Cluster. 

Author: ```suggestion
(_Single Instance_, _Master-Slave_, _Active Failover_ or Cluster).
```
should cluster also be italic, as in _Cluster_? Do we consider it as term coined by us?

Reviewer: yes, I did not include DC2DC here because it is not supported for win

> Please refer to the [_Deployment_](../Deployment/README.md) chapter for further information

Author: redundant
```suggestion
Please refer to the [_Deployment_](../Deployment/README.md) chapter for details.
```

> on deployment.

Author: ```suggestion
```
